README.md

```diff
@@ -160,12 +174,11 @@ mremi_contact:
         name:              contact_form
         validation_groups: [Default]
         subject_provider:  mremi_contact.subject_provider.noop
-        captcha_disabled:  false
-        captcha_type:      genemu_captcha # or genemu_recaptcha
+        captcha_type:      false
```

Comment:
This should be removed like the previous config you added. I think the captcha configuration must have a dedicated documentation after this one.